.golangci.yaml

```diff
@@ -0,0 +1,160 @@
+run:
+  deadline: 5m
+  skip-dirs:
+    - mock*
+  skip-files:
+    - "zz_generated.*\\.go$"
+    - ".*conversion.*\\.go$"
+linters:
+  disable-all: true
+  enable:
+    - asciicheck
+    - bodyclose
+    - dogsled
+    # - errcheck
+    # - errorlint
+    - exportloopref
+    # - goconst
+    # - gocritic
+    # - godot
+    # - gofmt
+    # - goimports
+    # - gosimple
+    # - govet
+    # - importas
+    # - gosec
+    # - ineffassign
+    # - misspell
+    # - nakedret
+    # - nilerr
+    # - noctx
+    # - nolintlint
+    # - prealloc
+    # - predeclared
+    # - revive
+    # - rowserrcheck
+    # - staticcheck
+    # - stylecheck
+    # - thelper
+    # - typecheck
+    # - unconvert
+    # - unused
+    # - whitespace
```

Comment:
These linters are commented out since the changes required in the code are overwhelming. I plan to enable them in batches so that its easy to change and also easy to review.

Reply:
Agreed. There are years of cruft to be fixed, easier to do and also to review in batches.